### PR TITLE
[sw/silicon_creator] Set watchdog bark threshold to 1.125 x bite threshold in ROM

### PR DESCRIPTION
**Pull request by alphan**

### CODE REVIEW

**Comment by alphan** [comment]
@cfrantz @moidx We enabled NMI and set the bark threhold to something proportional to the bite threshold stored in OTP in https://github.com/lowRISC/opentitan/commit/6ee4b6e5f11dce2f94ab193e67c45ccde7f4317b. This PR does the same for ROM. Do you think we need an OTP word for the bark threshold or a hardcoded fixed constant OK?

File: sw/device/silicon_creator/lib/drivers/watchdog_functest.c [hunk 1]
```diff
@@ -45,7 +51,11 @@ static rom_error_t watchdog_configure_disabled_test(void) {
   // Set watchdog bite threshold to 1ms.
   uint32_t threshold = 1 * udiv64_slow(kClockFreqAonHz, 1000, NULL);
```
Comment (dmcardle) [inline sw/device/silicon_creator/lib/drivers/watchdog_functest.c:52]: Aside: why does this multiply by 1?

**Review by dmcardle — APPROVED** [review]
LGTM!

File: sw/device/silicon_creator/lib/drivers/watchdog_functest.c (hunk 1 above)
Comment (alphan) [reply sw/device/silicon_creator/lib/drivers/watchdog_functest.c:52]: It looks like for consistency.

File: sw/device/silicon_creator/rom/rom_start.S [hunk 2]
```diff
@@ -26,10 +26,10 @@
 
 .equ UNIMP, 0xc0001073
 // We will configure the watchdog timers to reset the chip in the event that the
-// ROM stalls. The watchdog's bite and bark thresholds are set to 1 and 1.1
+// ROM stalls. The watchdog's bite and bark thresholds are set to 1 and 1.125
 // seconds, respectively, assuming a clock frequency of 200 kHz.
-.equ WDOG_BITE_THOLD, 0x30d400
-.equ WDOG_BARK_THOLD, WDOG_BITE_THOLD * 11 / 10
+.equ WDOG_BITE_THOLD, 0x30d40
```
[the remark this answers is not in the document]
Comment (alphan) [reply sw/device/silicon_creator/rom/rom_start.S:31]: I tried, but failed with 
```
[OPENOCD] Error: Check JTAG interface, timings, target power, etc.
[OPENOCD] Error: Trying to use configured scan chain anyway...
[OPENOCD] Error: riscv.tap: IR capture error; saw 0x1f not 0x01
[OPENOCD] Warn : Bypassing JTAG setup events due to errors
[OPENOCD] Error: failed write at 0x10, status=1
[OPENOCD] Info : [riscv.tap.0] Found 4 triggers
```

Could the bitstream with the fix not in the cache yet? I can try again later.

[the remark this answers is not in the document]
Comment (dmcardle) [reply sw/device/silicon_creator/rom/rom_start.S:31]: Weird! I just checked out this PR and the test suite passed (nice branch name, lol). Did you use `--define bitstream=gcp_splice`? I know that the bitstream for 0aa0b1cbfde8c0ad4646c90615404e0a1b5a3f3f is cached.

[the remark this answers is not in the document]
Comment (alphan) [reply sw/device/silicon_creator/rom/rom_start.S:31]: > Did you use `--define bitstream=gcp_splice`? 

Thanks for checking! Of course I didn't, duh 🤦🏼

**Pull request merged**